chef_master/source/data_collection.rst

```diff
@@ -0,0 +1,143 @@
+=====================================================
+Configure Data Collection
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/data_collection.rst>`__
+
+
+.. tag chef_automate_mark
+
+.. image:: ../../images/chef_automate_full.png
+   :width: 40px
+   :height: 17px
+
+.. end_tag
+
+Automatic Node Run Data Collection with Chef Server
+=======================================================
+
+.. note:: New in Chef 12.16. Chef 12.16.42 or greater and Chef Server 12.11.0 or greater are required.
+
+Nodes can send their run data to Chef Automate through the Chef server automatically. To enable this functionality, you must perform the following steps:
+
+ * Configure a Data Collector token in Chef Automate
+ * Configure your Chef server to point to Chef Automate
+
+Step 1: Configure a Data Collector token in Chef Automate
+------------------------------------------------------------
+
+All messages sent to Chef Automate are performed over HTTP and are authenticated with a pre-shared key called a ``token.`` Every Chef Automate installation configures a default token by default, but we strongly recommend that you create your own.
+
+To set your own token, add the following to your ``/etc/delivery/delivery.rb`` file:
+
+   .. code-block:: ruby
+
+      data_collector['token'] = 'sometokenvalue'
+      # Save and close the file
+
+To apply the changes, run:
+
+   .. code-block:: ruby
+
+      automate-ctl reconfigure
+
+
+If you do not configure a token, the default token value is: ``93a49a4f2482c64126f7b6015e6b0f30284287ee4054ff8807fb63d9cbd1c506``
+
+Step 2: Configure your Chef server to point to Chef Automate
```

Comment:
This section (ands those that follow it) all exist elsewhere - should we include dtags to ensure this gets updated?

Reply:
This replaces ``https://docs.chef.io/setup_visibility_chef_automate.html``.  Are they also elsewhere?

Reply:
Nevermind, I'm actually just thinking of the airgap guide - I can go back and do some tagging on that.